cmd/profilecli/upload.go

```diff
@@ -36,7 +36,7 @@ func addUploadParams(cmd commander) *uploadParams {
 	params.phlareClient = addPhlareClient(cmd)
 
 	cmd.Arg("path", "Path(s) to profile(s) to upload").Required().ExistingFilesVar(&params.paths)
-	cmd.Flag("extra-labels", "Add additional labels to the profile(s)").Default("job=profilecli-upload").StringMapVar(&params.extraLabels)
+	cmd.Flag("extra-labels", "Add additional labels to the profile(s)").StringMapVar(&params.extraLabels)
```

Comment:
the `job` label is not used in this context based on my discussion with @cyriltovena, but let me know if you have any concerns with this change